.gitignore

```diff
@@ -1,13 +1,14 @@
 *.pyc
-openedx_webhooks.db
-jira.pem
-jira.pub
-jira.uuid
-docs/_build
+.cache
 .coverage
-htmlcov
 .env
-.cache
+README.md
```

Comment:
Given that we have huge docsets written in ReST, and docstrings in ReST, and readme's in ReST, it might make sense to develop a ReST-centric workflow.